### PR TITLE
update(JS): web/javascript/reference/global_objects/promise/then

### PR DESCRIPTION
**Pull request by undead404**
Оригінальний вміст: [Promise.prototype.then()@MDN](https://developer.mozilla.org/en-us/docs/Web/JavaScript/Reference/Global_Objects/Promise/then), [сирці Promise.prototype.then()@GitHub](https://github.com/mdn/content/blob/main/files/en-us/web/javascript/reference/global_objects/promise/then/index.md)

Нові зміни:
- [mdn/content@b5c766f](https://github.com/mdn/content/commit/b5c766f4eecb4fcf9d8ba175caddb94f7c3e9d20)
- [mdn/content@6bd17cb](https://github.com/mdn/content/commit/6bd17cb9cbc2d11163617b9f71706e93fdd743c8)
- [mdn/content@c517363](https://github.com/mdn/content/commit/c51736321bee582320a6aed38f969e9f28a2fcd8)
- [mdn/content@82beb8d](https://github.com/mdn/content/commit/82beb8dba336c997fc91d0d54fe19227e119d1b1)
- [mdn/content@c1b574d](https://github.com/mdn/content/commit/c1b574d923c78f3fade2c920bd58bc7e3704dee8)
- [mdn/content@fcd80ee](https://github.com/mdn/content/commit/fcd80ee4c8477b6f73553bfada841781cf74cf46)
- [mdn/content@2eb202a](https://github.com/mdn/content/commit/2eb202adbe3d83292500ed46344d63fbbae410b5)
- [mdn/content@79f307c](https://github.com/mdn/content/commit/79f307ce6c91d661aa024addea2f6631f4853e47)
- [mdn/content@9a771cd](https://github.com/mdn/content/commit/9a771cdf71361a2d7528fa729e1512461a97db28)

### CODE REVIEW

**Comment by undead404** [comment]
Оновлено

**Comment by undead404** [comment]
Розв'язано конфлікт

**Comment by undead404** [comment]
Оновлено

**Comment by undead404** [comment]
Оновлено

File: uk_spelling_additions.txt [hunk 1]
```diff
@@ -304,6 +307,8 @@ x-висоту
 фронтенді
 фронтенду
 функцією-виконавцем
+функцію-виконавця
```
Comment (AdriandeCita) [inline uk_spelling_additions.txt:310]: Схоже, це дублікат
```suggestion
```

**Review by AdriandeCita — CHANGES_REQUESTED** [review]
Дрібничка

**Review by AdriandeCita — APPROVED** [review]

**Pull request merged**